### DISCUSSION TITLE
Portrait-aware Menubar?

### DISCUSSION THREAD

**Discussion by cmcconomy** [rxhanson/Rectangle#430] [Ideas]
I looked through the github Issues only to discover that the "thirds" function is actually vertical-aware! However, the icons accompanying the menubar drop-down are always drawn as landscape/horizontal representations of the "Rectangle" function (in other words, they are wider than they are tall).

Is it possible for a macos dropdown to be aware that it's being invoked on a monitor with a portrait aspect ratio, and display a set of Vertical Screen representation icons? This would make discoverability of the 'portrait-native' functionality easier!

(By the way - this app is outstanding! Amazing work! 🔥)

**Comment by rxhanson** [comment]
At first cut, I've rotated the icons just for thirds in v0.45, since these are the most common ones that people get confused about. It would be nice to have all of them change, but that's all I had time for at the moment (PRs welcome).

**Comment by cmcconomy** [comment]
Wow, this rules! Thank you so much.
